boards/others/candlelightfd/candlelightfd_stm32g0b1xx_dual.yaml

```diff
@@ -0,0 +1,15 @@
+identifier: candlelightfd/stm32g0b1xx/dual
+name: candleLightFD Dual
+type: mcu
+arch: arm
+toolchain:
+  - zephyr
+  - gnuarmemb
+ram: 144
+flash: 128
+supported:
+  - can
+  - gpio
+  - usb_device
+  - usbd
```

Comment:
`supported` section aims at defining features that could be tested against matching `tests/` and `samples/`.
Though, all these tests require a console output. Since no console is provided those tests can't even be built against this board, so whole `supported` section should be removed.